version/news/NEWS-2025.09.0-cucumberleaf-sunflower.md

```diff
@@ -109,17 +137,17 @@ After completing the package upgrade, carefully review the backed up files and t
 
 ### Dependencies
 
-- ([#8991](https://github.com/rstudio/rstudio-pro/issues/8991)): Nginx 1.29.1
+- (rstudio-pro#8991): Nginx 1.29.1
 - ([#15935](https://github.com/rstudio/rstudio/issues/15935)): Copilot Language Server 1.352.0
 - ([#15933](https://github.com/rstudio/rstudio/issues/15933)): Electron 37.2.6
 - ([#16062](https://github.com/rstudio/rstudio/issues/13924)): GWT 2.12.2
 - ([#15934](https://github.com/rstudio/rstudio/issues/15934)): Quarto 1.7.32
```

Comment:
```suggestion
- ([#15934](https://github.com/rstudio/rstudio/issues/15934)): Quarto 1.7.32
- (vscode-server#177): Code OSS for VSCode and Positron Pro sessions 1.102.0
- Node for VSCode and Positron Pro sessions 22.15.1
```

rstudio/vscode-server#177 isn't strictly about updating Code OSS but I believe it resolved the vulnerabilities in the issue and we don't have an issue that tracked updating code oss specifically AFAIK.